src/app/cube/details/details.component.html

```diff
@@ -90,10 +90,6 @@ <h2 class='section-title'>Reviews</h2>
             </cube-details-action-panel>
           </section>
           <section class='details__content-container__right-side-container'>
-            <h3 class='section-title'>Authors</h3>
```

Comment:
This needs to stay put. We need them on the details page but not on the cards.

Reply:
Ah I see, my bad